Dockerfile.assisted-service-onprem

```diff
@@ -38,4 +39,6 @@ COPY --from=config-gen $WORK_DIR/test_utils.py $WORK_DIR
 ENV WORK_DIR=$WORK_DIR
 
 COPY --from=assisted-service /assisted-service /assisted-service
+COPY --from=assisted-service /clients/* /clients/
```

Comment:
Why do you need it and why do you copy if from assisted-service that maybe not even built and not build it yourself?

Reply:
Looks like this was accidentally removed recently. Currently render_files.py uses the python bindings, but will be going away soon once the golang port of the script makes it in.